src/runner.rs

```diff
@@ -25,7 +25,48 @@ pub(crate) enum TickFlow {
     Exit,
 }
 
-pub(crate) const HELP_TEXT: &'static str = r"To start a transaction, enter 'start transaction' or 'begin'. To exit, enter 'exit' or press CTRL-D.";
+pub(crate) const HELP_TEXT: &'static str = r#"Shell Keys
+  Enter
+    - Runs the statement.
+  Escape+Enter (macOS)
+  Shift+Enter (Windows)
+    - Starts a new line to enter a statement that spans multiple lines. You can also copy input text with multiple lines and paste it into the shell.
```

Comment:
should we also include unix/linux? i unfortunately don't have a great way to test right now on a linux box, but i wonder what convention it will follow and whether we should mention it similar to how we mention macOS and Windows

Reply:
Yeah we should also include `*nix` here, the key binding is the same as macOS https://github.com/awslabs/amazon-qldb-shell/blob/4df3fb7b379fbf862445329d2b145b64e8411065/src/ui.rs#L160-L163 
Will update.

Reply:
Updated

Reply:
Updated